torch/_inductor/lowering.py

```diff
@@ -3868,16 +3867,56 @@ def truncdiv(a, b):
     return ops.truncdiv(a, b)
 
 
+@make_pointwise
+def logical_and(a, b):
+    return ops.logical_and(a, b)
+
+
+@make_pointwise
+def logical_not(a, b):
+    return ops.logical_not(a)
+
+
+@make_pointwise
+def logical_or(a, b):
+    return ops.logical_or(a, b)
+
+
+@make_pointwise
+def logical_xor(a, b):
+    return ops.logical_xor(a, b)
+
+
 @make_pointwise
 def bitwise_and(a, b):
     return ops.bitwise_and(a, b)
 
 
+@make_pointwise
+def bitwise_not(a):
+    return ops.bitwise_not(a)
+
+
 @make_pointwise
 def bitwise_or(a, b):
     return ops.bitwise_or(a, b)
 
 
+@make_pointwise
+def bitwise_xor(a, b):
+    return ops.bitwise_xor(a, b)
+
+
+@make_pointwise
+def bitwise_left_shift(a, b):
+    return ops.bitwise_left_shift(a, b)
+
+
+@make_pointwise
+def bitwise_right_shift(a, b):
+    return ops.bitwise_right_shift(a, b)
```

Comment:
You can simply assign to variables the result of `register_pointwise` below and then reuse them to register the in-place ones.

Reply:
Fixed.